client/gutenberg/extensions/mailchimp/view.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Internal dependencies
+ */
+import './view.scss';
+
+const blockClassName = 'wp-block-jetpack-mailchimp';
+
+const JetpackEmailSubscribe = {
+	validateEmail: function( email ) {
```

Comment:
😬 Something I try to avoid at all cost 🙈 

We already depend on [email-validator](https://www.npmjs.com/package/email-validator) in Calypso. I suspect it's a small enough dependency that you can pull it into the view without issue.

Reply:
If this works out it'd be great. I lifted the validation function directly from @artpi 's original implementation, so worth asking if there was anything non-standard about it? I'd assume the answer is no?

Reply:
06555f05bbf7c5a5fd8d7bc4f22e028a9c8ee9f9